### DISCUSSION TITLE
Rollback versions

### DISCUSSION THREAD

**Discussion by r2yj2y** [SnosMe/awakened-poe-trade#836] [Q&A]
Hey guys

How can i activate an older version of awakened?

Cheers

**Comment by toughguy247** [comment]
My awakened POE has been having issues every since I updated yesterday. Whenever I use it, it makes the entire rest of my screen black and lots of screen flickering. I have uninstalled, and reinstalled and still having the same issues. Really wish I could revert the update.

**Comment by Daeviling** [comment]
Same problem. Can't solve it on win 10. There was a post about writing "--disable-gpu" in object of Awaken Poe Trade, But it do nothing -_-

**Reply by Daeviling** [reply]
Actually it works . i'm blind and forgot one "-"

**Reply by toughguy247** [reply]
I tried the same thing, and yes indeed it works! So glad.
